public/app/themes/clarity/inc/amazon-s3-and-cloudfront-signing.php

```diff
@@ -48,14 +48,12 @@ public function __construct()
         $this->handlePageRequest();
 
         add_filter('http_request_args', function($args){
-            $headers = $args['headers'] ?? false;
-            if ($headers) {
-                $user_agent = $headers['user-agent'] ?? false;
+            if ($args['headers'] ?? false) {
+                $user_agent = $args['headers']['user-agent'] ?? false;
                 if ($user_agent === 'wp-offload-media') {
-                    $args['cookies'] = $this->createSignedCookie($this->cloudfront_url . '/*');
+                    $args['headers']['Authorization'] = 'Basic ' . $_ENV['BASIC_AUTH'];
```

Comment:
Hey, even if the secret looks like base64 when you run 

`kubectl -n $NSP get secret intranet-basic-auth -o yaml `

```
apiVersion: v1
data:
  auth: aW50cm...
```

k8s will decode it to user:xyz...

I think here $_ENV['BASIC_AUTH'] you might still need to run base64_encode.